plugins/GSdx/GSDrawScanlineCodeGenerator.x86.cpp

```diff
@@ -2889,7 +2889,7 @@ void GSDrawScanlineCodeGenerator::ReadTexel_SSE(int pixels, int mip_offset)
 				movdqa(ptr[&m_local.temp.test], xmm7);
 			}
 
-			for(int j = 0; j < 4; j++)
+			for(uint8 j = 0; j < 4; j++)
```

Comment:
This loop must be unrolled. It's only 4 iterations. If compiler fails to do so, when it can be unrolled by hand.